app/Console/Commands/HelperNotificationsCron.php

```diff
@@ -75,18 +76,24 @@ public function handle()
                             $events[$help->activity->event->id]->help[] = $helpInfo;
                         }
 
-                        $handledHelps[] = $help;
+                        if(!in_array($help->id, $handledHelpIds)) $handledHelpIds[] = $help->id;
                     }
                 }
+
+                unset($committee);
```

Comment:
Just curious what this is for?

Reply:
As we're doing this for many, many users and many, many committees with many, many relations, I became a bit concerned about memory usage. `unset` simply _unsets_ the variable and lets PHP's garbage collector know this variable is no longer needed. :)